packages/babel-plugin-emotion/src/utils/label.js

```diff
@@ -9,11 +14,23 @@ const sanitizeLabelPart = (labelPart: string) =>
 function getLabel(
   identifierName?: string,
   autoLabel: boolean,
-  labelFormat?: string,
+  labelFormat?: string | (LabelFormatOptions => string),
   filename: string
 ) {
   if (!identifierName || !autoLabel) return null
-  if (!labelFormat) return sanitizeLabelPart(identifierName)
+
+  const sanitizedName = sanitizeLabelPart(identifierName)
+
+  if (!labelFormat) {
+    return sanitizedName
+  }
+
+  if (typeof labelFormat === 'function') {
+    return labelFormat({
+      name: sanitizedName,
+      filename
```

Comment:
Could we call this `path` to distinguish it from the filename used in the string form of labelFormat?

Reply:
Sure, changed.